presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/RubixConfig.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive.rubix;
+
+import io.airlift.configuration.Config;
+
+public class RubixConfig
+{
+    boolean rubixEnabled;
+    boolean parallelWarmupEnabled;
+    String cacheLocation = "/tmp";
+
+    public boolean isRubixEnabled()
+    {
+        return rubixEnabled;
+    }
+
+    public boolean isParallelWarmupEnabled()
+    {
+        return parallelWarmupEnabled;
+    }
+
+    @Config("hive.rubix-enabled")
```

Comment:
This should be `hive.cache.enabled`. The fact that it uses Rubix under the covers is an implementation detail that doesn't need to be exposed to users and admins (similar, for example, to how we expose "http server" properties and not "jetty server" properties).

Reply:
done